services/src/test/scala/cromwell/pipeline/service/ProjectServiceTest.scala

```diff
@@ -108,5 +108,28 @@ class ProjectServiceTest extends AsyncWordSpec with Matchers with MockitoSugar {
         projectService.getUserProjectById(projectId, userId).failed.map { _ shouldBe ProjectNotFoundException() }
       }
     }
+
+    "getUserProjectByName" should {
+      val project1 = TestProjectUtils.getDummyProject()
+      val project2 = TestProjectUtils.getDummyProject(name = project1.name)
+
+      "return project with corresponding user id" taggedAs Service in {
+        when(projectRepository.getProjectsByName(project2.name)).thenReturn(Future.successful(Seq(project1, project2)))
+        projectService.getUserProjectByName(project2.name, project2.ownerId).map { _ shouldBe project2 }
+      }
+
+      "return project by name" taggedAs Service in {
```

Comment:
I think at this place we shold write `return project**s** by name`

Reply:
Thanks for response, but here we should return only one project at Services level, therefore  i wrote "byName".